build.rs

```diff
@@ -42,6 +42,10 @@ INCLUDE device.x"#
         f
     };
 
+    // Write an empty `sections.x` file to be overwritten by the application if desired.
+    let mut f_sections = File::create(out.join("sections.x")).unwrap();
+    f_sections.write(b"\n").unwrap();
+
```

Comment:
Are you sure it works? I investigated this a couple of months ago and found that there is no guaranteed order for search directories. With custom `sections.x` you can get either it or this empty `sections.x` included.